tasks/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "rust-bitcoin-maintainer-tools"
```

Comment:
```suggestion
name = "bitcoin-maintainer-tools"
```

Reply:
You could be right. Also the 'rust-bitcoin' bit refers to the github org kinda too.